src/iop/clipping.c

```diff
@@ -2324,9 +2324,24 @@ void gui_post_expose(struct dt_iop_module_t *self, cairo_t *cr, int32_t width, i
     snprintf(dimensions, sizeof(dimensions), "%.0fx%.0f", (float)procw * g->clip_w, (float)proch * g->clip_h);
```

Comment:
As we are at it, can you add spaces around x :  "%.0f x %.0f"

This will be even more readable I think. Thanks.

Reply:
Sure.  And while we're at that, what about using a Unicode "Multiplication X" character instead of the regular lower-case x?  So:  **1920 ✕ 1080**

https://www.fileformat.info/info/unicode/char/2715/index.htm

I think that would look even nicer, and the center of the X would mark the exact center of the crop window if we're careful to use e.g. "%5.0f ✕ %-5.0f".

Any objection to using that Unicode "Multiplication X"?

Reply:
Yes, sadly last time I have introduced an Unicode character in dt it was an issue for Windows. So I would prefer a plan x for now until this is fixed (maybe need another font) on Windows.

Reply:
Yep, I had that concern too.  Fwiw though, iop/clipping.c does already use a unicode character...
`pango_layout_set_text(layout, "ꝏ", -1); `
... though I do not know whether it renders correctly in Windows.  ;-)

The commit I'm adding to this pull req just sticks with the plain old lower-case "x".

Reply:
FYI, these symbols aren't rendered for me on Linux, at least in a web browser. So please don't use such characters, it's not just about Windows.